src/planner/binder/statement/bind_export.cpp

```diff
@@ -8,11 +8,36 @@
 #include "duckdb/main/database.hpp"
 #include "duckdb/common/file_system.hpp"
 #include "duckdb/planner/operator/logical_set_operation.hpp"
+#include "duckdb/parser/parsed_data/exported_table_data.hpp"
 
 #include <algorithm>
 
 namespace duckdb {
 
+//! Sanitizes a string to have only low case chars and underscores
+string SanitizeExportIdentifier(const string &str) {
+	// Copy the original string to result
+	string result(str);
+
+	for (idx_t i = 0; i < str.length(); ++i) {
+		auto c = str.at(i);
```

Comment:
`str.at(i)` introduces bounds checks on every access, since we are only accessing until str.length() we know the bounds cannot be exceeded, so you can just use `str[i]`